nsv13/game/areas.dm

```diff
@@ -27,6 +27,204 @@
+	icon_state = "smaint"
+
+/area/maintenance/nsv/deck3/frame3/central
+	name = "Deck 3 Frame 3 Central Maintenance"
+	icon_state = "maintcentral"
+
+/area/maintenance/nsv/deck3/frame4/central
+	name = "Deck 3 Frame 4 Central Maintenance"
+	icon_state = "maintcentral"
+
+/area/hallway/nsv/deck2/frame1/port
+	name = "Deck 2 Frame 1 Port Hallway"
+	icon_state = "hallF"
+
+/area/hallway/nsv/deck2/frame1/starboard
+	name = "Deck 2 Frame 1 Starboard Hallway"
+	icon_state = "hallF"
+
+/area/hallway/nsv/deck2/frame2/port
+	name = "Deck 2 Frame 2 Port Hallway"
+	icon_state = "hallP"
+
+/area/hallway/nsv/deck2/frame2/starboard
+	name = "Deck 2 Frame 2 Starboard Hallway"
+	icon_state = "hallS"
+
+/area/hallway/nsv/deck2/frame3/port
+	name = "Deck 2 Frame 3 Port Hallway"
+	icon_state = "hallC"
+
+/area/hallway/nsv/deck2/frame3/starboard
+	name = "Deck 2 Frame 3 Starboard Hallway"
+	icon_state = "hallC"
+
+/area/hallway/nsv/deck2/frame4/port
+	name = "Deck 2 Frame 4 Port Hallway"
+	icon_state = "hallP"
+
+/area/hallway/nsv/deck2/frame4/starboard
+	name = "Deck 2 Frame 4 Starboard Hallway"
+	icon_state = "hallS"
+
+/area/hallway/nsv/deck2/frame4/central
+	name = "Deck 2 Frame 4 Central Hallway"
+	icon_state = "hallC"
+
+/area/hallway/nsv/deck2/frame5/port
+	name = "Deck 2 Frame 5 Port Hallway"
+	icon_state = "hallP"
+
+/area/hallway/nsv/deck2/frame5/starboard
+	name = "Deck 2 Frame 5 Starboard Hallway"
+	icon_state = "hallS"
+
+/area/hallway/nsv/deck2/frame5/central
+	name = "Deck 2 Frame 5 Central Hallway"
+	icon_state = "hallC"
+
+/area/hallway/nsv/deck3/frame1/port
+	name = "Deck 3 Frame 1 Port Hallway"
+	icon_state = "hallP"
+
+/area/hallway/nsv/deck3/frame1/starboard
+	name = "Deck 3 Frame 1 Starboard Hallway"
+	icon_state = "hallS"
+
+/area/hallway/nsv/deck3/frame1/central
+	name = "Deck 3 Frame 1 Central Hallway"
+	icon_state = "hallC"
+
+/area/hallway/nsv/deck3/frame2/port
+	name = "Deck 2 Frame 2 Port Hallway"
+	icon_state = "hallP"
+
+/area/hallway/nsv/deck3/frame2/starboard
+	name = "Deck 2 Frame 2 Starboard Hallway"
+	icon_state = "hallS"
+
+/area/hallway/nsv/deck3/frame3/port
+	name = "Deck 3 Frame 3 Port Hallway"
+	icon_state = "hallP"
+
+/area/hallway/nsv/deck3/frame3/starboard
+	name = "Deck 3 Frame 3 Starboard Hallway"
+	icon_state = "hallS"
+
+/area/hallway/nsv/deck3/frame3/central
+	name = "Deck 3 Frame 3 Central Hallway"
+	icon_state = "hallC"
+
+/area/hallway/nsv/deck3/frame4/port
+	name = "Deck 3 Frame 4 Port Hallway"
+	icon_state = "hallP"
+
+/area/hallway/nsv/deck3/frame4/starboard
+	name = "Deck 3 Frame 4 Starboard Hallway"
+	icon_state = "hallS"
+
+/area/hallway/nsv/deck1/hallway
+	name = "Deck 1 Primary Hallway"
+	icon_state = "hallC"
+
+/area/crew_quarters/nsv/observation
+	name = "Observation Lounge"
+	icon_state = "Sleep"
+
+/area/crew_quarters/nsv/officerquarters
+	name = "Bridge Officer's Quarters"
+	icon_state = "Sleep"
+
+/area/nsv/hanger/deck3/starboard
+	name = "Deck Three Starboard Hanger Bay"
```

Comment:
Inconsistent naming procedure using letters for some and numbers for others.

Reply:
Resolved by d30a32d